src/CapabilityService.WebApi/Startup.cs

```diff
@@ -48,6 +50,32 @@ public void ConfigureServices(IServiceCollection services)
                 .SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
             
             services.AddCorrelationId();
+
+            services.Configure<ApiOptions>(Configuration.GetSection(ApiOptions.Position));
+
+            services.AddCors(options =>
+            {
+	            var apiOptions = new ApiOptions();
+	            Configuration.GetSection(ApiOptions.Position).Bind(apiOptions);
+	            var originHosts = new ArrayList(new[] {"https://build.dfds.cloud", "https://backstage.dfds.cloud" });
+
+	            options.AddDefaultPolicy(builder =>
+	            {
+		            if (apiOptions.Cors != null && apiOptions.Cors.Origin != null)
+		            {
+			            originHosts.Add(apiOptions.Cors.Origin);
```

Comment:
It might be a good idea is we could load in multiple hosts via the config?

Reply:
Added!